Lib/fontTools/designspaceLib/convert5to4.py

```diff
@@ -0,0 +1,538 @@
+                    filename=source.filename,
+                    path=source.path,
+                    font=source.font,
+                    name=source.name,
+                    location=_filter_location(axes_with_range, source.location),
+                    layerName=source.layerName,
+                    familyName=source.familyName,
+                    styleName=source.styleName,
+                    # copyLib=False,
+                    # copyInfo=False,
+                    # copyGroups=False,
+                    # copyFeatures=False,
+                    muteKerning=source.muteKerning,
+                    muteInfo=source.muteInfo,
+                    mutedGlyphNames=source.mutedGlyphNames,
+                )
+            )
+
+        # Instances: same as Sources + compute missing names
+        for instance in doc.instances:
+            if not location_in_selection(instance.location, region_selection):
+                continue
+
+            vf_instance_names = _make_STAT_names(doc, instance, ribbi_mapping)
+            vf_doc.addInstance(
+                InstanceDescriptor(
+                    filename=instance.filename,
+                    path=instance.path,
+                    font=instance.font,
+                    name=instance.name,
+                    location=_filter_location(axes_with_range, instance.location),
+                    familyName=(
+                        instance.familyName or vf_instance_names.familyName or None
+                    ),
+                    styleName=instance.styleName or vf_instance_names.styleName or None,
+                    postScriptFontName=(
+                        instance.postScriptFontName
+                        or vf_instance_names.postScriptFontName
+                        or None
+                    ),
+                    styleMapFamilyName=(
+                        instance.styleMapFamilyName
+                        or vf_instance_names.styleMapFamilyName
+                        or None
+                    ),
+                    styleMapStyleName=(
+                        instance.styleMapStyleName
+                        or vf_instance_names.styleMapStyleName
+                        or None
+                    ),
+                    # localisedFamilyName=instance.localisedFamilyName
+                    # or vf_instance_names.localisedFamilyName,
+                    # localisedStyleName=instance.localisedStyleName
+                    # or vf_instance_names.localisedStyleName,
+                    # localisedStyleMapFamilyName=instance.localisedStyleMapFamilyName
+                    # or vf_instance_names.localisedStyleMapFamilyName,
+                    # localisedStyleMapStyleName=instance.localisedStyleMapStyleName
+                    # or vf_instance_names.localisedStyleMapStyleName,
+                    # Deprecated
+                    # glyphs=None,
+                    # kerning=True,
+                    # info=True,
```

Comment:
same here, what's the purpose of commented out lines? if any, write accompanying comment